src/Maker/MakeMiddleware.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/**
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ *
+ * @internal
+ */
+final class MakeMiddleware extends AbstractMaker
+{
+    public static function getCommandName(): string
+    {
+        return 'make:middleware';
```

Comment:
I think it needs to be `make:messenger-middleware`. While the only concept in Symfony of a "middleware" is the Messenger middleware, the concept, in general, is generic.